src/CmsManager/CmsManagerInterface.php

```diff
@@ -46,6 +46,16 @@ public function findContainer($name, PageInterface $page, BlockInterface $parent
      */
     public function getPageByUrl(SiteInterface $site, $slug);
 
+    /**
+     * Returns a fully loaded page ( + blocks ) from a custom url.
+     *
+     * @param SiteInterface $site
+     * @param string        $slug
+     *
+     * @return PageInterface
+     */
+    public function getPageByCustomUrl(SiteInterface $site, $slug);
```

Comment:
This is a BC break :(

You can't add new methods to an interface with a minor release

Reply:
> This is a BC break :(
> 
> You can't add new methods to an interface with a minor release

Oh, should i add this to master branch?

Reply:
You can comment the method, with a hint to uncomment it with the next major release